suite/tests/client-interface/drwrap-test-callconv.dll.cpp

```diff
@@ -190,13 +190,17 @@ dr_init(client_id_t id)
 
 #ifdef PLATFORM_HAS_THISCALL
     thiscall = DRWRAP_CALLCONV_THISCALL;
+#elif defined(RISCV64)
+    thiscall = DRWRAP_CALLCONV_RISCV_LP64;
```

Comment:
The default should be this already: this change should be removed and the default changed instead.  Ditto below.

Reply:
Done